### DISCUSSION TITLE
iOS key signing help

### DISCUSSION THREAD

**Discussion by simondhearn** [codemagic-ci-cd/codemagic-docs#1489] [Q&A] [answered]
Hi,

I'm new to Codemagic and iOS development. I followed the instruction for providing keys for iOS building on the following video
https://docs.codemagic.io/flutter-code-signing/ios-code-signing/

I went to App Store Connect, created a key, as suggested, and downloaded the key, this was a .p8 file. I then went to 

Codemagic->teams->personal account->Code signing identities->iOS certificates(upload a certificate file)

It states load a .p12, I did try to upload my .p8 file and it completely ignored my file..

I really don't know how to progress, I'm probably doing something really stupid but please I would appreciate any help,thank you

Simon

**Comment by icarusdust** [comment, marked as answer]
Hi @simondhearn, the link you sent is for signing iOS apps with the Workflow Editor, please note that code signing identities is only for codemagic.yaml. Check [this](https://docs.codemagic.io/yaml-code-signing/signing-ios/), if you want to use code signing identities. Otherwise, you can follow the steps in the doc you attached,

**Reply by simondhearn** [reply]
thank you
